config/webpack.config.js

```diff
@@ -191,23 +196,20 @@ const configGenerator = (buildOptions, apps) => {
       new ManifestPlugin({
         fileName: 'file-manifest.json',
       }),
+      new webpack.SourceMapDevToolPlugin(
+        isOptimizedBuild
+          ? {
+              append: `\n//# sourceMappingURL=${bucket}/generated/[url]`,
+              filename: '[file].map',
+            }
+          : {},
+      ),
```

Comment:
When passed an empty object, it defaults to inlining the source maps. Since the source maps are useful for us in every environment, I think it makes sense to load this plugin every time.

[Docs](https://webpack.js.org/plugins/source-map-dev-tool-plugin/)

> filename (string): Defines the output filename of the SourceMap (will be inlined if no value is provided).

Reply:
Does this change the type of inline sourcemap? There are several different variations: https://webpack.js.org/configuration/devtool/

Reply:
Ah man, per the [example](https://webpack.js.org/plugins/source-map-dev-tool-plugin/#basic-use-case) it looks like it defaults to `inline-source-map`. Looks like `EvalSourceMapDevToolPlugin` is what we want for dev builds.